package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "hardhat-project",
+  "name": "blindex",
   "scripts": {
     "prepare": "husky install",
-    "test": "hardhat --network mainnetFork test:dir --deploy-fixture ./test/integration/",
-    "test:gas": "cross-env REPORT_GAS=true hardhat --network mainnetFork test:dir --deploy-fixture ./test/integration/",
+    "test": "npm run deploy:fork:reset && hardhat --network mainnetFork test:dir --deploy-fixture ./test/integration/",
```

Comment:
what if someone has a local environment he'd like to run the tests?

Reply:
For the tests to pass, at least how they are written today, you'll have to reset the local environment with this script anyway or they'll fail.

We could work on changing it, but not worth the trouble now and the reason I've made this change is in order to save us commands to run the tests (for example I've forgotten the need of it since it has been a while and wasted some time on it)

Reply:
In any case, if it will be needed in the future it will be very easy adding it as a separate script